pkg/vmprovider/providers/vsphere/vmprovider_vm_test.go

```diff
@@ -235,9 +235,9 @@ func vmTests() {
 					var o mo.VirtualMachine
 					Expect(vcVM.Properties(ctx, vcVM.Reference(), nil, &o)).To(Succeed())
 					Expect(o.Summary.Config.NumCpu).To(BeEquivalentTo(vmClass.Spec.Hardware.Cpus))
-					Expect(o.Summary.Config.NumCpu).To(Not(BeEquivalentTo(configSpec.NumCPUs)))
+					Expect(o.Summary.Config.NumCpu).ToNot(BeEquivalentTo(configSpec.NumCPUs))
```

Comment:
I really wish Gomega only supports one matcher to keep things consistent instead of 3 for the same case (e.g. `ToNot`, `NotTo`, `To(Not)`)...